homeassistant/components/starlink/dish_status.py

```diff
@@ -0,0 +1,21 @@
+"""Defines the API response when querying the addon for Starlink metrics."""
+from dataclasses import dataclass
+
+
+@dataclass
+class DishyStatus:
```

Comment:
I'm not entirely sure why this dataclass is needed? `get_status` already seems to return an object that can be used/passed instead?

Reply:
It does return an object, but there's no type hints (yet). dataclass is there just so I get nice autocomplete outside coordinator